lib/sonata/tip/executeUserTip.ts

```diff
@@ -19,7 +19,7 @@ async function executeUserTip({
   if (amount <= 0) throw Error('Invalid amount');
 
   const post = await getCastByHash(postHash);
-  const recipientFid = post.authorFid;
+  const recipientFid = post.author.fid;
```

Comment:
is this always the case?
- sometimes is `post.authorFid` present while your new updates are missing?